Error when running make install - missing ProgressBar method

When I try to run make install, it outputs the following error:
```
./provisioner.go:466:37: cannot use Ui literal (type *Ui) as type packer.Ui in return argument:
        *Ui does not implement packer.Ui (missing ProgressBar method)
make: *** [Makefile:4: prepare] Error 2
```
Probably because recently packer added a stackable progressbar to ui.go

https://github.com/hashicorp/packer/commit/541c68aed556052da94119d8ef710375210e9d94

--- a/README.md
+++ b/README.md
@@ -1,7 +1,7 @@
 Packer InSpec Provisioner
 =======
 
-[![Build Status](http://img.shields.io/travis/jrbeilke/packer-provisioner-inspec.svg)][travis]
+[![Build Status](https://img.shields.io/travis/jrbeilke/packer-provisioner-inspec/master.svg)][travis]
 
 [travis]: https://travis-ci.org/jrbeilke/packer-provisioner-inspec
 

--- a/adapter_test.go
+++ b/adapter_test.go
@@ -24,7 +24,7 @@ func TestAdapter_Serve(t *testing.T) {
 
 	config := &ssh.ServerConfig{}
 
-	ui := new(ui)
+	ui := new(packer.NoopUi)
 
 	sut := newAdapter(done, &l, config, newUi(ui), communicator{})
 	go func() {
@@ -91,36 +91,6 @@ func (a addr) Network() string {
 
 func (a addr) String() string {
 	return "test"
-}
-
-type ui int
-
-func (u *ui) Ask(s string) (string, error) {
-	*u++
-	return s, nil
-}
-
-func (u *ui) Say(s string) {
-	*u++
-	log.Println(s)
-}
-
-func (u *ui) Message(s string) {
-	*u++
-	log.Println(s)
-}
-
-func (u *ui) Error(s string) {
-	*u++
-	log.Println(s)
-}
-
-func (u *ui) Machine(s1 string, s2 ...string) {
-	*u++
-	log.Println(s1)
-	for _, s := range s2 {
-		log.Println(s)
-	}
 }
 
 type communicator struct{}

--- a/main.go
+++ b/main.go
@@ -1,7 +1,7 @@
 package main
 
 import (
-	"github.com/mitchellh/packer/packer/plugin"
+	"github.com/hashicorp/packer/packer/plugin"
 )
 
 func main() {

--- a/provisioner.go
+++ b/provisioner.go
@@ -497,3 +497,7 @@ func (ui *Ui) Machine(t string, args ...string) {
 	ui.ui.Machine(t, args...)
 	<-ui.sem
 }
+
+func (ui *Ui) ProgressBar() packer.ProgressBar {
+	return new(packer.NoopProgressBar)
+}

